sound/soc/sof/pm.c

```diff
@@ -282,6 +296,12 @@ int snd_sof_suspend(struct device *dev)
 }
 EXPORT_SYMBOL(snd_sof_suspend);
 
+int snd_sof_pre_remove(struct snd_sof_dev *sdev)
+{
+	return sof_pre_remove(sdev);
+}
+EXPORT_SYMBOL(snd_sof_pre_remove);
+
```

Comment:
Both these functions are unnecessary. Why not just export sof_send_pm_ctx_ipc() and call it from the remove() callbacks for each platform

Reply:
I exported new function snd_sof_dsp_power_down_notify(), this encapsulates sof_send_pm_ctx_ipc(). Are you OK with that?